zerver/models.py

```diff
@@ -752,6 +752,21 @@ def major_tos_version(self):
         else:
             return -1
 
+class Team(models.Model):
+    team_name = models.CharField(max_length=100)
+    members = models.ManyToManyField(UserProfile, through='TeamMembership')
+    realm = models.ForeignKey(Realm)
+
+    class Meta:
+        unique_together = (('team_name', 'realm'),)
```

Comment:
Swap the order of these; I think that'll result in a better database index structure :)

Reply:
Done.